wordpress.org/public_html/wp-content/plugins/plugin-directory/readme/class-parser.php

```diff
@@ -273,8 +275,14 @@ protected function parse_readme_contents( $contents ) {
 			$this->tags = explode( ',', $headers['tags'] );
 			$this->tags = array_map( 'trim', $this->tags );
 			$this->tags = array_filter( $this->tags );
-			$this->tags = array_diff( $this->tags, $this->ignore_tags );
-			$this->tags = array_slice( $this->tags, 0, 5 );
+			if ( array_intersect( $this->tags, $this->ignore_tags ) ) {
+				$this->tags = array_diff( $this->tags, $this->ignore_tags );
+				$this->warnings['ignored_tags'] = true;
+			}
+			if ( count( $this->tags ) > 6 ) {
```

Comment:
I guess it should be `5` instead of `6` here.

Reply:
6 is correct, since it's actually 0 indexed.  It's limited to 6 tags (0, 1, 2, 3, 4, 5 indexes)

Reply:
But this will never give more than 5 tags. `array_slice( $this->tags, 0, 5 )` Do we allow 5 or 6 tags?

Reply:
The code is designed to handle a scenario where there are more than 6 elements in the `$this->tags` array. If this condition is met, it trims the array to only include the first 5 elements and sets a warning flag `too_many_tags` to indicate that there were originally too many tags in the array. Hope that make sense now?

Reply:
Yeah.. I misread the code..

We've documented it as being 12 tags everywhere else, and I'm 85% certain this was supposed to be 6 tags.. but 5 is even better.

Reply:
```suggestion
			if ( count( $this->tags ) > 5 ) {
```